example-config.el

```diff
@@ -27,4 +27,7 @@
 
 (load-theme 'doom-snazzy t)
 
+;; Opt-in to load `custom.el' after `config.el'; you may also
+;; customize the option using the customization UI.
```

Comment:
With the current state of the conditional load of the custom file, this is actual a lie.

I couldn't figure out how to get the values of customization variables, other than their state. I didn't want to make the conditional very long.

Reply:
With #112 setting up to always load the `custom-file`, this could be "To opt-out use this code: (setq rational-load-custom-file nil)"